google/data_source_container_registry.go

```diff
@@ -0,0 +1,105 @@
+package google
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceGoogleContainerRepo() *schema.Resource {
```

Comment:
I would split the 2 data sources in different files to follow our convention of one resource or one data source per source file

Reply:
Done.